docker/dev/Dockerfile

```diff
@@ -15,15 +15,15 @@ ENV ENABLE_FIND_FASTEST_APT_SOURCE=${ENABLE_FIND_FASTEST_APT_SOURCE}
 
 COPY docker/scripts /install/scripts
 
-RUN bash /install/scripts/find-fastest-apt.sh && \
-    apt-get install -y git curl vim libssl-dev wget unzip openjdk-11-jdk && \
-    bash /install/scripts/install-cmake.sh && \
-    bash /install/scripts/install-bazel.sh && \
-    bash /install/scripts/install-python.sh
+RUN bash /install/scripts/find-fastest-apt.sh
+RUN apt-get install -y git curl vim libssl-dev wget unzip openjdk-11-jdk
+RUN bash /install/scripts/install-cmake.sh
+RUN bash /install/scripts/install-bazel.sh
+RUN bash /install/scripts/install-python.sh
 
 ARG DEVICE=cu110
-RUN bash /install/scripts/install-tensorrt.sh "${DEVICE}"
-RUN bash /install/scripts/patch-cuda.sh "${DEVICE}"
+RUN bash /install/scripts/install-tensorrt.sh
+RUN bash /install/scripts/install-cudnn.sh
```

Comment:
why not use an Nvidia Docker with the specific cudnn ?

Reply:
The Nvidia Docker's CuDNN is 8.0, which is too lower for EasyCV.